### PR TITLE
docs: add opentelemetry guidelines to README

### PR DESCRIPTION
**Pull request by annafang-google**
Added opencensus to opentelemetry migration instruction in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,6 +379,51 @@ func main() {
...
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"go.opencensus.io/trace"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/bridge/opencensus"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"google.golang.org/api/option"
+)
+
+func main() {
+	// Be careful about using trace.AlwaysSample in a production application
```
Comment (jackwotherspoon) [inline README.md:401]:
```suggestion
	// Be careful about using trace. AlwaysSample in a production application
```

File: README.md [hunk 2]
```diff
@@ -379,6 +379,51 @@ func main() {
...
+	// with significant traffic: a new trace will be started and exported for
```
Comment (jackwotherspoon) [inline README.md:402]: nit: I find this comment block a bit confusing...

**Review by jackwotherspoon — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (annafang-google) [reply README.md:402]: // trace.AlwaysSample() is expensive. Recommend replacing it with your own 
    // sampler for your production.
How does this look?

Comment (jackwotherspoon) [reply README.md:402]: maybe a slight tweak?

```
// trace.AlwaysSample() is expensive. Replacing it with your own 
// sampler for production environments is recommended.
```

**Review by jackwotherspoon — APPROVED** [review]
LGTM, just left the one small comment to update

**Pull request merged**